src/main/scala/is/hail/distributedmatrix/BlockMatrix.scala

```diff
@@ -507,10 +548,12 @@ class BlockMatrix(val blocks: RDD[((Int, Int), BDM[Double])],
     require(this.cols <= Integer.MAX_VALUE)
     val cols = this.cols.toInt
 
-    def seqOp(a: Array[Double], p: (Int, Array[Double])): Array[Double] = p match { case (offset, v) =>
-      System.arraycopy(v, 0, a, offset, v.length)
-      a
+    def seqOp(a: Array[Double], p: (Int, Array[Double])): Array[Double] = p match {
+      case (offset, v) =>
```

Comment:
orthogonal to this PR, but when there's only one case, is `val (offset, v) = p` preferable?